### PR TITLE
51/feature/style rec page

### PR DESCRIPTION
**Pull request by seabelis**
Makes the json human-friendly.

### CODE REVIEW

File: bestbook/static/build/css/style.css [hunk 1]
```diff
@@ -85,9 +85,7 @@ select:focus, select:active, input[type=text]:focus, input[type=text]:active {
 }
 
 .block {
```
Comment (jamesachamp) [inline bestbook/static/build/css/style.css:87]: This class is used in several templates.  It may be better to revert these changes, create a new class with the `border-bottom` rule, and replace the `block` class in `browse.html` with the new class.

**Review by jamesachamp — CHANGES_REQUESTED** [review]

File: bestbook/static/build/css/style.css [hunk 2]
```diff
@@ -85,9 +85,7 @@ select:focus, select:active, input[type=text]:focus, input[type=text]:active {
...
-    border: 5px solid #ccd6dd;
-    border-radius: 15px;
-
+    border-bottom: 1px dashed #233045;
     width: 100%;
     margin: 0 auto;
 }
```
Comment (jamesachamp) [inline bestbook/static/build/css/style.css:91]: This class is used in several templates.  It may be better to revert these changes, create a new class with the `border-bottom` rule, and replace the `block` class in `browse.html` with the new class.

**Review by mekarpeles — APPROVED** [review]
lgtm
